authentication/README.md

```diff
@@ -0,0 +1,83 @@
+# Azure AI Content Safety Authentication
+
+Each request to an Azure Cognitive Service must include an authentication header. This header passes along a subscription key or authentication token, which is used to validate your subscription for a service or group of services.
+
```

Comment:
Let's put the original doc link here.
https://learn.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell